src/main/java/com/faforever/client/game/MissingGamePathNotifier.java

```diff
@@ -40,7 +40,7 @@ public void onMissingGamePathEvent(MissingGamePathEvent event) {
     List<Action> actions = Collections.singletonList(
         new Action(i18n.get("missingGamePath.locate"), chooseEvent -> eventBus.post(new GameDirectoryChooseEvent()))
     );
-
-    notificationService.addNotification(new PersistentNotification(i18n.get("missingGamePath.notification"), Severity.WARN, actions));
+    String message = i18n.get("missingGamePath.notification");
+    notificationService.addNotification(new ImmediateNotification(message, message, Severity.WARN, actions));
```

Comment:
The client aims to be as non-intrusive as possible. Immediate notifications should therefore be used sparingly and only in cases where a user's immediate action is required.

Let's say someone only has the client installed for chatting in #aeolus, or maybe he's a client developer who doesn't actually have the game installed but starts it very often. This user gets an annoying notification now, every time he starts the client.

This is why we made it a PersistentNotification.

Reply:
Good point. 
I'll change that back to persistent, but I think we need a very interrupting popup when the player hits the "play" button without a game path set. The persistent notification is easily overlooked and the current client does not give any direct feedback on pressing the "play" button without a game path set: [Ladder1v1Controller.java](https://github.com/FAForever/downlords-faf-client/blob/develop/src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java#L172)
I'm not sure if there are other places with the same problem